scenario.py

```diff
@@ -77,24 +78,38 @@ def __init__(self, params, experiment_path):
         self.contributivity_list = []
 
         # Number of epochs and mini-batches in ML training
-        if 'epoch_count' in params.keys():
+        if 'epoch_count' in params:
             self.epoch_count = params['epoch_count']
             assert(self.epoch_count > 0)
         else:
             self.epoch_count = 40
         
-        if 'minibatch_count' in params.keys():
-            self.minibatch_count = params['minibatch_count']
-            assert(self.minibatch_count > 0)
-        else:
-            self.minibatch_count = 20
-
-        if 'minibatch_count' in params.keys():
+        if 'minibatch_count' in params:
             self.minibatch_count = params['minibatch_count']
             assert self.minibatch_count > 0
         else:
             self.minibatch_count = 20
-
+        
+        # Contributivity methods
+        methods_default = [
+            "Shapley values",
+            "Independant scores",
+            "TMCS values"
+        ]
+        
+        self.methods = []
+        if not 'methods' in params:
+            self.methods = methods_default
+        else:
+            if params['methods']:
+                for el in params['methods']:
+                    if el in methods_default:
+                        self.methods.append(el)
+                    else:
+                        raise Exception('method ' + el + ' is not in methods list.')
+            else:
+                raise Exception("No contributivity method given in config file")
```

Comment:
What do you think of something like this:
```
ALL_METHODS_LIST =  = [...]
DEFAULT_METHODS_LIST = [...]

if 'methods' in params:

	for method in params['methods']:
			if method in ALL_METHODS_LIST:
				self.methods.append(method)
			else:
				raise Exception('method ' + method + ' is not in methods list: ' + ALL_METHODS_LIST)
				
else:
	self.methods = DEFAULT_METHODS_LIST
```

I choose to put 2 methods list (DEFAULT and ALL) because I guess we might not want to run all methods by default.

Reply:
I did the same on the IS-method PR